js/client/modules/@arangodb/testutils/instance-manager.js

```diff
@@ -956,13 +969,53 @@ class instanceManager {
     return shutdownSuccess;
   }
 
+  detectAgencyAlive() {
+    let count = 20;
+    while (count > 0) {
+      let haveConfig = 0;
+      let haveLeader = 0;
+      let leaderId = null;
+      for (let agentIndex = 0; agentIndex < this.agencyConfig.agencySize; agentIndex ++) {
+        let reply = this.agencyConfig.agencyInstances[agentIndex].getAgent('/_api/agency/config', 'GET');
+        if (this.options.extremeVerbosity) {
+          print("Response ====> ");
+          print(reply);
+        }
+        if (!reply.error && reply.code === 200) {
+          let res = JSON.parse(reply.body);
+          if (res.hasOwnProperty('lastAcked')){
+            haveLeader += 1;
+          }
+          if (res.hasOwnProperty('leaderId') && res.leaderId !== "") {
+            haveConfig += 1;
+            if (leaderId === null) {
+              leaderId = res.leaderId;
+            } else if (leaderId !== res.leaderId) {
+              haveLeader = 0;
+              haveConfig = 0;
+            }
+          }
+        }
+        if (haveLeader === 1 && haveConfig === this.agencyConfig.agencySize) {
+          print("Agency Up!");
```

Comment:
Can we remove this again?

Reply:
lets do this after macos is stable again.